api/client/proto/authservice.proto

```diff
@@ -1433,4 +1453,12 @@ service AuthService {
     //
     // Recovery approved tokens last RecoveryApprovedTokenTTL.
     rpc ApproveAccountRecovery(ApproveAccountRecoveryRequest) returns (types.UserTokenV3);
+
+    // ChangeAuthnFromAccountRecovery (exclusive to cloud users) is the third step in account
+    // recovery, after RPC's StartAccountRecovery and ApproveAccountRecovery. This step sets a new
+    // password or adds a new mfa device, allowing the user to regain access to their account with
+    // the new credentials. When the new authentication is successfully set, all user lock is
+    // removed so the user can login immediately after they complete the recovery process.
+    rpc ChangeAuthnFromAccountRecovery(ChangeAuthnFromAccountRecoveryRequest)
```

Comment:
I really think I should name this `CompleteAccountRecovery`, b/c at this step, you change your authentication, so after success a user can login with the new credentials right afterwards. To me this marks the end of account recovery.

The reason why I held back or wasn't sure is this endpoint does not give you new recovery codes and in the UI, the `flow` ends with receiving new recovery codes (depicted below), which is a separate endpoint. But technically user can skip this step and still log in, they just won't get new recovery codes unless they generate a new one from their logged in dashboard. 

**UI recover second factor flow:**
start -> approve -> complete -> view mfa list and optionally delete some -> new recovery codes

**UI recover password:**
start -> approve -> complete -> new recovery codes

Reply:
Sounds good.

Reply:
renamed to `CompleteAccountRecovery`